File: app/src/main/java/com/geridea/trentastico/utils/NumbersUtils.java
package com.geridea.trentastico.utils;


/*
 * Created with ♥ by Slava on 31/03/2017.
 */

public class NumbersUtils {

    public static int compare(long x, long y) {
        return (x < y) ? -1 : ((x == y) ? 0 : 1);
    }

}

File: app/src/main/java/com/geridea/trentastico/utils/NumbersUtils.kt
package com.geridea.trentastico.utils


/*
 * Created with ♥ by Slava on 31/03/2017.
 */

object NumbersUtils {

    fun compare(x: Long, y: Long): Int {
        return if (x < y) -1 else if (x == y) 0 else 1
    }

}
